src/PatchworkProtocol.sol

```diff
@@ -359,8 +366,42 @@ contract PatchworkProtocol {
         if (newOwner == address(0)) {
             revert ScopeTransferNotAllowed(address(0));
         }
-        s.owner = newOwner;
-        emit ScopeTransfer(scopeName, msg.sender, newOwner);
+        s.ownerElect = newOwner;
```

Comment:
@robdoesstuff think it is worth adding events for election and cancels?

Reply:
I think the rule of thumb is that any mutate should result in an event so yes I'll add those